portality/settings.py

```diff
@@ -475,7 +475,7 @@
         "journal,article" : {
             "auth" : False,
             "role" : None,
-            "query_validator" : "public_query_validator",
+            # "query_validator" : "public_query_validator",
```

Comment:
Relating to my above comment, since you are not using the public_query_validator here you shouldn't need to update the `public_query_validator` function.